R/qcMetric_MSMS.R

```diff
@@ -197,7 +197,7 @@ current study. ",
       return(list(plots = lpl, qcScores = qc_score))
     }, 
     qcCat = "Prep", 
-    qcName = "MSMS:~%s", 
+    qcName = "MSMS:~Missed~Cleavages~%s", 
```

Comment:
caution:
in lines 194-195 the `%s` is replaced by `MC` or `MC~Var`. Thus this now reads `MSMS:~Missed~Cleavages~MC` etc..

So, maybe revert line 200 to original and do:
```
colnames(qc_score)[colnames(qc_score) == "valMC"] = sprintf(.self$qcName, "Missed~Cleavages")
colnames(qc_score)[colnames(qc_score) == "valMCVar"] = sprintf(.self$qcName, "Missed~Cleavages~Var")
```